### ISSUE TITLE
Cant create/mount any volumes, safe reports "Unexpected Error Occurred"

### ISSUE THREAD

**Issue by stevecrye** [safeapp/safe#52]
Was excited to try Safe because it seemed a perfect solution for portable encryption of files on a USB stick, because the safe.exe is self-contained.  But, I can't get it to create/mount anything. Have tried on C: NTFS, external Sandisk USB FAT32. Always get the same error. It reports the volume was created, then fails on mount. My computer is Win7 x64 sp1. I run Classic Shell to avoid the annoyance of the Win7 GUI. 

BTW, I did not let it run the recommended security changes. I don't want to encrypt my page file until I know Safe will work.
Thanks,
Steve

![safefail1](https://cloud.githubusercontent.com/assets/2738227/6191257/43a93304-b365-11e4-8feb-dfd27cf9404b.jpg)

**Comment by rianhunter** [comment]
Safe currently requires you make the changes for it to function. A future version won't, but for now that was the only way to ensure local encryption. Thanks for trying it!

**Comment by stevecrye** [comment]
Thanks for the quick reply, Ryan; 

Just a bit concerned about such a deep change to the system (pagefile encryption). Worried it might cause instability or slowness. 

Steve 


Stephen T. Crye | Network Infrastructure Manager 
stcrye@episd.org | 915-230-2611 (rings all my phones) | 915-412-5769 (Google Voice) 
The El Paso Independent School District | Technology Services | 6531 Boeing Drive | El Paso TX 79925 


----- Original Message -----

**Comment by rianhunter** [comment]
Yup that's fair, lots of other people have echoed the same concern (including myself!)

In general, page file encryption is a must if you're serious about security. FWIW I run it on all my systems (as well as full disk encryption) and i haven't noticed significant slowdown. This would make sense as IO requests are usually disk bound (cpu is way way faster compared to time waiting for data to be copied off the disk).
